src/test/java/com/sphy/hotelmanagementapplication/controller/HotelControllerTest.java

```diff
@@ -241,7 +241,7 @@ void findHotelByName() throws Exception {
         when(hotelService.getHotelByName(anyString())).thenReturn(hotelDTO);
 
         // Return
-        mockMvc.perform(get("/api/hotelName/hotelName"))
+        mockMvc.perform(get("/api/hotelName/hotelName", "hotelName"))
```

Comment:
add {} to hotelName inside the string "/api/hotelName/hotelName", and i would personally change the hotelName there to name, so that it is clear to everyone that it is a placeholder value, and not the actual string we want to check (""/api/hotelName/{name}"")